project-base/app/tests/App/Functional/Model/Order/OrderFacadeEditTest.php

```diff
@@ -45,6 +45,7 @@ protected function setUp(): void
     {
         parent::setUp();
 
+        $this->setInputPriceTypeToInputPriceTypeWithVat();
```

Comment:
do not we want another test for input price type without VAT?

Reply:
I did not feel the need, do you?